app/src/main/res/layout/widget_camera.xml

```diff
@@ -13,26 +13,35 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:visibility="invisible"
+        android:layout_gravity="center"
         android:adjustViewBounds="true"
         android:contentDescription="@string/widget_camera_contentdescription"
         android:scaleType="fitCenter" />
-    <ImageView
+
+    <FrameLayout
         android:id="@+id/widgetCameraPlaceholder"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_marginStart="4dp"
-        android:layout_marginTop="4dp"
-        android:layout_marginBottom="4dp"
-        android:src="@drawable/app_icon_round"
-        android:contentDescription="@string/widget_camera_contentdescription"
-        android:scaleType="fitCenter" />
+        android:layout_gravity="center"
+        android:background="@android:color/transparent"
+        android:layout_height="match_parent">
+
+        <ImageView
+            android:layout_width="48dp"
+            android:layout_height="48dp"
+            android:layout_gravity="center"
+            android:contentDescription="@string/widget_camera_contentdescription"
+            android:scaleType="fitCenter"
+            android:src="@android:drawable/ic_menu_camera" />
+    </FrameLayout>
+
     <ImageView
         android:id="@+id/widgetCameraError"
-        android:contentDescription="@string/widget_entity_fetch_error"
         android:layout_width="16dp"
         android:layout_height="16dp"
-        android:layout_marginBottom="4dp"
+        android:layout_gravity="start|bottom"
+        android:layout_marginBottom="25dp"
+        android:layout_marginStart="12dp"
```

Comment:
What's up with this alignment? It looks even more random than the current one, also because of the different values. We also try to stick to an 8dp grid.
![Screenshot_20240917-175430.png](https://github.com/user-attachments/assets/f48a1f45-188a-4548-95c5-2dc70612e569)